packages/server/lib/plugins/child/run_plugins.js

```diff
@@ -1,4 +1,3 @@
-// @ts-check
 // this module is responsible for loading the plugins file
```

Comment:
For some reason this caused errors to be raised when using `ts-node/esm`, I don't quite know why, so I just removed the `@ts-check`. These files are not very type safe anyway, I don't think the `ts-check` is doing too much with the very lax `tsconfig.json` in `packages/server`, anyway. We can work on making it nice and strict in the future.